web_console/odc-about-developer-perspective.adoc

```diff
@@ -5,6 +5,10 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective.
+[NOTE]
+====
+The default web console perspective is shown depending on the user's role. (If the user is recognised as a developer, a Developer perspective will be selected by default.)
```

Comment:
user's role>the role of the user
Remove the parenthesis for the second sentence.
a Developer perspective will be selected by default.> the **Developer** perspective is displayed by default.

Reply:
```suggestion
The default web console perspective is shown depending on the role of the user. If the user is recognised as a developer, the *Developer* perspective is displayed by default.
```